matching-game/tests/components/button.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Button } from '@/components/button/button'
+
+describe('Button', () => {
+  it('Renders the correct text', () => {
```

Comment:
`it` is for actually making a test. the syntax is `it(testDescription: string, test: function)`